app/models/intake/ctc_intake.rb

```diff
@@ -287,6 +287,18 @@ class Intake::CtcIntake < Intake
     self.spouse_ssn = self.spouse_ssn.remove(/\D/) if spouse_ssn_changed? && self.spouse_ssn
   end
 
+  before_save do
```

Comment:
I would rather this be a `before_validation` hook. That allows you to change the data as early as possible, which means the transformed data will also be what any validation sees. Not a blocker.